workspaces/liferay-sample-workspace/client-extensions/liferay-sample-captcha/src/main/java/com/liferay/sample/CaptchaValidationRestController.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.sample;
+
+import com.liferay.client.extension.util.spring.boot3.BaseRestController;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * @author Manuele Castro
+ */
+@RequestMapping("/captcha/validation")
+@RestController
+public class CaptchaValidationRestController extends BaseRestController {
+
+	public CaptchaValidationRestController(
+		RestTemplateBuilder restTemplateBuilder) {
+
+		_restTemplate = restTemplateBuilder.build();
+	}
+
+	@PostMapping
+	public ResponseEntity<String> post(
+		@AuthenticationPrincipal Jwt jwt, @RequestBody String json) {
+
+		log(jwt, _log, json);
+
+		JSONObject jsonObject = new JSONObject(json);
+
+		MultiValueMap<String, String> body = new LinkedMultiValueMap<>();
+
+		body.add("secret", "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe");
```

Comment:
Hey @manuelecastro and @daiazius, Can you please try to use the org.springframework.beans.factory.annotation.Value annotation to avoid having the secret on the code directly?

Thx